integration/tools/validation/src/main/java/alluxio/cli/fuse/CorrectnessValidationOperation.java

```diff
@@ -0,0 +1,35 @@
+package alluxio.cli.fuse;
```

Comment:
The file names are redundant, i will prefer to use package name for those information instead of calling all the files Correctness Validation XXX.

Reply:
cli/fuse/datacorrectness/ ?
or maybe there are better names